customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,90 @@
+from re import S
+import streamlit as st
+from inference.predict import predict 
+from PIL import Image
+import pandas as pd 
+
+import pickle 
+def main(): 
+    st.title("End to End Customer Satisfaction Pipeline with ZenML")
+
+
+    image = Image.open('_assets/high_level_overview.png')
+    st.image(image, caption='High Level Pipeline')
+
+    st.markdown("""
+    #### Description of Features 
+    This app is designed to predict the customer satisfaction score for a given customer.
+
+    * Payment Sequential:-   Customer may pay an order with more than one payment method. If he does so, a sequence will be created to accommodate all payments.
+    * Payment Installments:- Number of installments chosen by the customer.
+    * Payment Value:-       Total amount paid by the customer.
+    * Price:-              Price of the product.   
+    * Frieght Value:-      Frieght value of the product.   
+    * Product Name length:- Number of characters extracted from the product name.
+    * Product Description length:- Number of characters extracted from the product description.
+    * Product photos Quantity:- Number of product published photos
+    * Product weight measured in grams:- Weight of the product measured in grams.
+    * Product length (CMs):- product length measured in centimeters.
+    * Product height measured in centimeters:- Height of the product measured in centimeters.
+    * Product width (CMs):- Width of the product measured in centimeters.
```

Comment:
Since this is markdown, maybe you can specify a markdown table as it would make it neater / smarter.

Reply:
Sure, it is added, you can check in the website, it's looking nice :)